### PR TITLE
Update why-design-for-bitcoin.md

### PR DESCRIPTION
**Pull request by pavlenex**
Co-Authored-By: @sqcken
Related #202

### CODE REVIEW

File: guide/getting-started/why-design-for-bitcoin.md [hunk 1]
```diff
@@ -37,35 +37,37 @@ Illustration sources
...
+## Build what you want
 
-The idea that there are no gatekeepers is baked into the foundation of Bitcoin and its ecosystem. A Bitcoin wallet designed by you is as official as one created by a big corporation.
+Bitcoin is permissionless; this means that there are no individual, corporate, or governmental gatekeepers deciding what’s allowed, so anything that you build has the potential to grow beyond what even the most well-funded can imagine.
 
-## Blaze the trail
+## Blaze trails one pixel at a time
 
-There are still very few dedicated designers working in the Bitcoin ecosystem. As we have seen with the evolution of the internet, it takes time to adopt new technology. If you are curious and excited about new challenges, here's your chance.
+Few open designers currently work full-time on Bitcoin. Most have 9–5 jobs, and like bitcoin’s most open-source developers, contribute when they can. As the ecosystem grows and more investors enter the space, more opportunities to be one of the first full-time bitcoin designers are bound to open up.
```
Comment (GBKS) [inline guide/getting-started/why-design-for-bitcoin.md:52]:
```suggestion
Few open designers currently work full-time on Bitcoin. Most have 9–5 jobs, and like most of Bitcoin’s open-source developers, contribute when they can. As the ecosystem grows, more opportunities to be among the first full-time Bitcoin designers are bound to open up.
```
I'd remove `investors`, as that's not the only funding source for open-source work and the sentence works without it as well.

File: guide/getting-started/why-design-for-bitcoin.md [hunk 2]
```diff
@@ -37,35 +37,37 @@ Illustration sources
...
+
+So if you’re looking for the next big thing in design, you can stop.
```
Comment (GBKS) [inline guide/getting-started/why-design-for-bitcoin.md:54]:
```suggestion
So if you’re looking for the next big thing in design, you found it.
```
"You can stop" sounded dismissive, like "you're not going to find it".

File: guide/getting-started/why-design-for-bitcoin.md [hunk 3]
```diff
@@ -37,35 +37,37 @@ Illustration sources
...
 
 ## Design for your needs
 
-Many bitcoin projects start by identifying an unmet need. Unlike traditional finance, the barriers to entry are shallow. So if you were ever unhappy with the service of your bank or financial provider, here's your chance to shape something better.
+Many design projects start by identifying an unmet need then designing something that meets it. Unlike traditional finance, the barriers to entry into bitcoin are shallow. So if you feel like your needs aren’t being met by your bank or service provider, here’s your chance to shape something better.
```
Comment (GBKS) [inline guide/getting-started/why-design-for-bitcoin.md:58]:
```suggestion
Many design projects start by identifying an unmet need then designing something that meets it. Unlike traditional finance, the barriers to entry into Bitcoin are shallow. So if you feel like your needs aren’t being met by your bank or service provider, here’s your chance to shape something better.
```

File: guide/getting-started/why-design-for-bitcoin.md [hunk 4]
```diff
@@ -37,35 +37,37 @@ Illustration sources
...
 
 ## Work in public
 
-Bitcoin is open-source, and designing for Bitcoin allows you also work on the same ideas of publicly sharing and collaborating on design information. Free from agencies and corporations' structures, open collaboration allows every participant to have a voice and shape the way they work based on their interests and preferences. Find out more about [open design]({{ '/guide/getting-started/open-design/' | relative_url }}).
+Bitcoin development is open-source. Designing for bitcoin is no different. It lets you learn, grow, and exchange ideas in the same open and collaborative way. Learn out more about bitcoin’s [open design]({{ '/guide/getting-started/open-design/' | relative_url }}) philosophy.
```
Comment (GBKS) [inline guide/getting-started/why-design-for-bitcoin.md:62]:
```suggestion
Bitcoin development is open-source. Designing for Bitcoin is no different. It lets you learn, grow, and exchange ideas in the same open and collaborative way. Learn out more about the [open design]({{ '/guide/getting-started/open-design/' | relative_url }}) philosophy.
```
Open design is not exclusive to Bitcoin.

File: guide/getting-started/why-design-for-bitcoin.md [hunk 5]
```diff
@@ -37,35 +37,37 @@ Illustration sources
...
 
 ## Design money
 
-Money is a powerful force and therefore heavily regulated and controlled. The composition of bitcoin allows us to trust in its value and availability while also providing vast freedom to experiment and build on top of it. The emergence of thousands of so-called alternative coins, hundreds of wallets, and the many bitcoin businesses are the most apparent signifier of just how open this space is.
+Central authorities tightly control Government-issued currencies. This means that they are anything but open and experimental. At every level, bitcoin places a great deal of value on openness and experimentation, letting designers build the future of money in their own way. The emergence of thousands of “alternative” coins, hundreds of wallets, and countless Bitcoin businesses signify just how open this space is.
```
Comment (GBKS) [inline guide/getting-started/why-design-for-bitcoin.md:66]:
```suggestion
Central authorities tightly control Government-issued currencies. This means that they are anything but open and experimental. At every level, Bitcoin places a great deal of value on openness and experimentation, letting designers build the future of money in their own way. The emergence of thousands of “alternative” coins, hundreds of wallets, and countless Bitcoin businesses signify just how open this space is.
```

File: guide/getting-started/why-design-for-bitcoin.md [hunk 6]
```diff
@@ -37,35 +37,37 @@ Illustration sources
...
 
 ## Join a global community
 
-The doors are wide open, and Bitcoin design is discussed publicly online and any social media service you can think of. Design and build with friends and strangers, participate in the various communities and help shape an ecosystem for design and creativity.
+Design and build experiences with friends and strangers. Participate in or even help lead one of the fastest growing design ecosystems in the world.
```
Comment (GBKS) [inline guide/getting-started/why-design-for-bitcoin.md:70]: `one of the fastest growing design ecosystems in the world` I know this is just meant to sound impressive, but I find it a but fluffy and would prefer something that is more welcoming and about community.

Comment (pavlenex) [reply guide/getting-started/why-design-for-bitcoin.md:70]: What's more welcoming than participating and lead in one of the fastest-growing design ecosystems?

Comment (GBKS) [reply guide/getting-started/why-design-for-bitcoin.md:70]: Is it one of the fastest-growing design ecosystems? I see way more design stuff happening in other ecosystems.

**Review by GBKS — APPROVED** [review]

**Pull request merged**